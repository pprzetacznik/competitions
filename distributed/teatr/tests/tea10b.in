100000000 3 2 1 1000000 273824060420851171
