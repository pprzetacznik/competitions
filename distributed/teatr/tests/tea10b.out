1874997040199202
